npm-publisher-patch.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Get the name of the current Git branch and then ensure that only
+# the master Git branch is deployable to NPM by this script.
+branch=$(git branch | sed -n -e 's/^\* \(.*\)/\1/p')
+
+# Only push this update to the NPM if the current branch is the master branch
+if [ "$branch" == "master" ]
+then
+  echo "Building and deploying the $branch branch to NPM..."
+
+  yarn build:prod && npm version patch && npm publish --access public
```

Comment:
Should it be configurable which type of version you're publishing? This won't always be patch versions, right?

Reply:
It should, but I have no idea how to proceed

Reply:
You could make this a parameter to this script?

Then you could use `./npm-publisher.sh patch` to do a patch release?

Reply:
Something like this should do the trick to get the parameter:

```
    updateType=$1

    if [[ "$updateType" == "" ]]
    then
        updateType="patch"
    fi
```